plugin/storage/integration/integration.go

```diff
@@ -453,9 +453,12 @@ func (s *StorageIntegration) testGetLatestProbability(t *testing.T) {
 	defer s.cleanUp(t)
 
 	s.SamplingStore.InsertProbabilitiesAndQPS("newhostname1", samplemodel.ServiceOperationProbabilities{"new-srv3": {"op": 0.123}}, samplemodel.ServiceOperationQPS{"new-srv2": {"op": 11}})
-	s.SamplingStore.InsertProbabilitiesAndQPS("dell11eg843d", samplemodel.ServiceOperationProbabilities{"new-srv": {"op": 0.1}}, samplemodel.ServiceOperationQPS{"new-srv": {"op": 4}})
+	s.SamplingStore.InsertProbabilitiesAndQPS("dell11eg843d", samplemodel.ServiceOperationProbabilities{"new-srv": {"op": 0.23}}, samplemodel.ServiceOperationQPS{"new-srv": {"op": 4}})
+	s.SamplingStore.InsertProbabilitiesAndQPS("sghdvavdasvd", samplemodel.ServiceOperationProbabilities{"new-srv": {"op": 0.2}}, samplemodel.ServiceOperationQPS{"new-srv": {"op": 8}})
+	s.SamplingStore.InsertProbabilitiesAndQPS("sjsvdfgvdsavg", samplemodel.ServiceOperationProbabilities{"new-srv": {"op": 0.1}}, samplemodel.ServiceOperationQPS{"new-srv": {"op": 2}})
+	s.SamplingStore.InsertProbabilitiesAndQPS("newjaegerv2", samplemodel.ServiceOperationProbabilities{"new-srv": {"op": 0.106}}, samplemodel.ServiceOperationQPS{"new-srv": {"op": 9}})
```

Comment:
what are you trying to achieve with these additional inserts?

Reply:
Previously, there were only two tests, and the code is this:
``` go
lengthOfSearchResult := len(searchResult.Hits.Hits)
if lengthOfSearchResult == 0 {
	return nil, nil
}
hit := searchResult.Hits.Hits[lengthOfSearchResult-1]
```
With the two test cases, it was sure that the test would not fail. That's why I added more inserts. To increase my surety.

If you want, I can remove it.

Reply:
unless there is an extra edge condition triggered, how is 6 inserts different from 2?

Reply:
You are right. 2 inserts function the same as 6 inserts. Removed in the current push